src/Lazy.tsx

```diff
@@ -54,13 +45,24 @@ export const Lazy: React.FC<{
   /**
    * This is used to control when children are mounted
    */
-  const [canMount, setCanMount] = React.useState(!!startMounted.value)
+  const [canMount, setCanMount] = React.useState(false)
   /**
    * Ensure we don't mount after the component has been unmounted
    */
   const isSelfMounted = React.useRef(true)
 
-  const opacity = useSharedValue(cancelLazyFadeIn || startMounted.value ? 1 : 0)
+  /**
+   * We start mounted if we are the focused tab, or if props.startMounted is true.
+   */
+  const shouldStartMounted =
+    typeof _startMounted === 'boolean'
+      ? _startMounted
+      : focusedTab.value === name
```

Comment:
It seems that reading `.value` in render is not recommended, according to: https://x.com/kzzzf/status/1805222107011317861

This is not your fault though, the code previously did this too.